README-Windows.md

```diff
@@ -8,11 +8,16 @@ is related to the lack of formal support for Zeek on Windows. We got it
 working with [our own Zeek port](https://github.com/brimsec/zeek/tree/master/brim/windows),
 but the best we could do in our first attempt uses 32-bit
 [Cygwin](https://www.cygwin.com/). We'll be working to improve this over time.
-2. We don't yet have the necessary Windows certifications in place to "sign"
-the release (like we do for macOS) and hence provide a warning-free
-install experience. Your anti-virus software is likely to jump in with "Are
-you sure you want to run this?" warnings and scans for viruses. Rest assured
-that we've not put viruses in our code, but we understand if you're concerned
-about installing under such conditions. You may want to install on a scratch
-Windows VM if that feels safer, or watch the repo to be notified when we've
-secured the certifications in an upcoming release.
+
+2. We sign the Windows installer we create with every new Brim release;
+   however, you may still see a Microsoft Defender SmartScreen popup when you
```

Comment:
```suggestion
however, you may still see a Microsoft Defender SmartScreen popup when you
```
Content is 👍, but a nit: Were these three leading spaces buying us anything here? It renders the same with and without them in my local `grip`, so it seems we might as well drop them since it makes the text version look more tidy.

Reply:
👍